Scripts/s4ap/persistance/ap_session_data_store.py

```
@@ -164,9 +183,15 @@ def get_career(self) -> str:
     def get_slot(self) -> int:
         return self._get_value(S4APSettings.SLOT)
 
-    def _get_value(self, key: str) -> Any:
-        generic_settings_data_store: S4APGenericDataStore = self._data_manager.get_generic_settings_data()
-        return generic_settings_data_store.get_value_by_key(key)
+    def _get_value(self, key: str, default=None) -> Any:
+        try:
+            generic_settings_data_store: S4APGenericDataStore = self._data_manager.get_generic_settings_data()
+            value = generic_settings_data_store.get_value_by_key(key)
+            return value if value is not None else default
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)